stdlib/public/core/Integers.swift.gyb

```diff
@@ -2271,7 +2271,8 @@ extension BinaryInteger {
 /// other arithmetic methods and operators.
 public protocol FixedWidthInteger :
   BinaryInteger, LosslessStringConvertible
-  where Magnitude : FixedWidthInteger
+  where Magnitude : FixedWidthInteger & UnsignedInteger,
+        Stride : FixedWidthInteger & SignedInteger
```

Comment:
@stephentyrone I'm referring to the line above.  Either "`& SignedInteger`" is redundant—in which case you should drop it—or you're constraining `Stride` to be signed here.  Or, I'm missing something ;-)

Reply:
What we get from `Strideable` is `SignedNumeric`. `SignedInteger` is a *very* tiny refinement of `BinaryInteger, SignedNumeric`, but I think it's the correct requirement semantically.

Reply:
Okay then!